appserver/admingui/core/src/main/resources/org/glassfish/admingui/core/Strings.properties

```diff
@@ -150,6 +150,8 @@ tree.accessLog=Access Log
 tree.clusters=Clusters
 tree.standaloneInstances=Stand-Alone Instances
 
+tree.support=Support
```

Comment:
Is there a reason you couldn't put this in the currently empty Strings.properties file in the support module?